.github/workflows/test.yml

```diff
@@ -3,11 +3,10 @@ on:
   pull_request:
   push:
     branches:
-      - master
-      - "releases/*"
+      - main
```

Comment:
`master` is also used here: https://github.com/ros-tooling/action-ros-ci/blob/1ff2c804b4c2383146d8cd8444dfda64ab4bf7ac/.github/workflows/automerge.yml#L5